R/phlawd_db_maker.R

```diff
@@ -3,3 +3,60 @@
 library(tidyverse)
 library(RSQLite)
 library(rentrez)
+library(XML)
+
+# historically, ids were session numbers for safety reasons
+r_search <- entrez_search(db="nuccore", term="labridae[ORGN] AND cytb[GENE]")
```

Comment:
Please see https://style.tidyverse.org/ for how to style R code. In particular you should always use `<-` for assignment and when passing parameters via `=` there needs to be space on either side.